pkgs/top-level/all-packages.nix

```diff
@@ -13594,6 +13586,19 @@ in
 
   lyx = callPackage ../applications/misc/lyx { };
 
+  magnetophonDSP = {
```

Comment:
now you do want to group them like this? 
https://github.com/NixOS/nixpkgs/pull/17215#issuecomment-235880114